doc/en/reference.rst

```diff
@@ -1010,6 +1010,23 @@ loaded.
 This is not meant to be set by users, but is set by pytest internally with the name of the current test so other
 processes can inspect it, see :ref:`pytest current test env` for more information.
 
+.. envvar:: PY_COLORS
+
+When set to "1", pytest will use color in terminal output.
+When set to "0", pytest will not use color.
```

Comment:
Would this be clearer?

```suggestion
When set to ``"1"``, pytest will use color in terminal output.
When set to ``"0"``, pytest will not use color.
```

# Before

![image](https://user-images.githubusercontent.com/1324225/87016606-99c25280-c1d7-11ea-8019-89b1cc7e2317.png)

# After

![image](https://user-images.githubusercontent.com/1324225/87016736-c4141000-c1d7-11ea-82f9-c183b3d77b66.png)

Reply:
Yes thanks!